s3_file_field/_multipart_boto3.py

```diff
@@ -16,13 +16,22 @@ def __init__(self, storage: 'S3Boto3Storage'):
         self._client: s3.Client = resource.meta.client
         self._bucket_name: str = storage.bucket_name
 
-    def _create_upload_id(self, object_key: str) -> str:
+    def _create_upload_id(
+        self,
+        object_key: str,
+        content_type: str = None,
+        content_disposition: str = None,
+    ) -> str:
+        metadata = {}
```

Comment:
Let's rename `metadata` to `boto3_kwargs` or something, just because there's actually a separate kwarg to `create_multipart_upload` called `Metadata`, for setting metadata on the object in S3.